website/src/components/brand-picker/brand-picker.js

```diff
@@ -91,7 +91,6 @@ export const BrandPicker = () => {
 					<BrandCard brand={brands.STG} />
 					<BrandCard brand={brands.BOM} />
 					<BrandCard brand={brands.BSA} left={[0, 0, 0, 4, 4]} />
```

Comment:
`left={[0, 0, 0, 4, 4]}` should be fine now with that Cell fix, wasn't handling 0's before